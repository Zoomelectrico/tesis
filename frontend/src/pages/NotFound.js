import React, { useEffect } from "react";
import { Row, Col, Container } from "reactstrap";
import { Navbar } from "../components";

const NotFound = props => {
  useEffect(() => {
    document.body.classList.add("bg-default");
    return function cleanup() {
      document.body.classList.remove("bg-default");
    };
  }, []);
  return (
    <div className="main-content">
      <Navbar {...props} />
      <div className="header bg-gradient-info py-7 py-lg-8">
        <Container>
          <Row className="justify-content-center">
            <Col md="7">
              <h2 className="text-center text-white">
                Error 404 <br /> No encontramos esta ruta{" "}
                <span role="img" aria-label="emoji">
                  ðŸ™…
                </span>
              </h2>
            </Col>
          </Row>
        </Container>
      </div>
    </div>
  );
};

export default NotFound;
