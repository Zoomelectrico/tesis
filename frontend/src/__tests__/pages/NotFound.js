import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../pages/NotFound';

describe('Not Found', () => {
  it('Render Correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
