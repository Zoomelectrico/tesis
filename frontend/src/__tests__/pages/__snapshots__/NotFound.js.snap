// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Render Correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Navigationbar />,
        <div
          className="header bg-gradient-info py-7 py-lg-8"
        >
          <Container
            tag="div"
          >
            <Row
              className="justify-content-center"
              tag="div"
            >
              <Col
                md="7"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <h2
                  className="text-center text-white"
                >
                  Error 404 
                  <br />
                   No encontramos esta ruta
                   
                  <span
                    aria-label="emoji"
                    role="img"
                  >
                    ðŸ™…
                  </span>
                </h2>
              </Col>
            </Row>
          </Container>
        </div>,
      ],
      "className": "main-content",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Container
            tag="div"
          >
            <Row
              className="justify-content-center"
              tag="div"
            >
              <Col
                md="7"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <h2
                  className="text-center text-white"
                >
                  Error 404 
                  <br />
                   No encontramos esta ruta
                   
                  <span
                    aria-label="emoji"
                    role="img"
                  >
                    ðŸ™…
                  </span>
                </h2>
              </Col>
            </Row>
          </Container>,
          "className": "header bg-gradient-info py-7 py-lg-8",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Row
              className="justify-content-center"
              tag="div"
            >
              <Col
                md="7"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <h2
                  className="text-center text-white"
                >
                  Error 404 
                  <br />
                   No encontramos esta ruta
                   
                  <span
                    aria-label="emoji"
                    role="img"
                  >
                    ðŸ™…
                  </span>
                </h2>
              </Col>
            </Row>,
            "tag": "div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Col
                md="7"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <h2
                  className="text-center text-white"
                >
                  Error 404 
                  <br />
                   No encontramos esta ruta
                   
                  <span
                    aria-label="emoji"
                    role="img"
                  >
                    ðŸ™…
                  </span>
                </h2>
              </Col>,
              "className": "justify-content-center",
              "tag": "div",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <h2
                  className="text-center text-white"
                >
                  Error 404 
                  <br />
                   No encontramos esta ruta
                   
                  <span
                    aria-label="emoji"
                    role="img"
                  >
                    ðŸ™…
                  </span>
                </h2>,
                "md": "7",
                "tag": "div",
                "widths": Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "Error 404 ",
                    <br />,
                    " No encontramos esta ruta",
                    " ",
                    <span
                      aria-label="emoji"
                      role="img"
                    >
                      ðŸ™…
                    </span>,
                  ],
                  "className": "text-center text-white",
                },
                "ref": null,
                "rendered": Array [
                  "Error 404 ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": "br",
                  },
                  " No encontramos esta ruta",
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "aria-label": "emoji",
                      "children": "ðŸ™…",
                      "role": "img",
                    },
                    "ref": null,
                    "rendered": "ðŸ™…",
                    "type": "span",
                  },
                ],
                "type": "h2",
              },
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Navigationbar />,
          <div
            className="header bg-gradient-info py-7 py-lg-8"
          >
            <Container
              tag="div"
            >
              <Row
                className="justify-content-center"
                tag="div"
              >
                <Col
                  md="7"
                  tag="div"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  <h2
                    className="text-center text-white"
                  >
                    Error 404 
                    <br />
                     No encontramos esta ruta
                     
                    <span
                      aria-label="emoji"
                      role="img"
                    >
                      ðŸ™…
                    </span>
                  </h2>
                </Col>
              </Row>
            </Container>
          </div>,
        ],
        "className": "main-content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Container
              tag="div"
            >
              <Row
                className="justify-content-center"
                tag="div"
              >
                <Col
                  md="7"
                  tag="div"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  <h2
                    className="text-center text-white"
                  >
                    Error 404 
                    <br />
                     No encontramos esta ruta
                     
                    <span
                      aria-label="emoji"
                      role="img"
                    >
                      ðŸ™…
                    </span>
                  </h2>
                </Col>
              </Row>
            </Container>,
            "className": "header bg-gradient-info py-7 py-lg-8",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Row
                className="justify-content-center"
                tag="div"
              >
                <Col
                  md="7"
                  tag="div"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  <h2
                    className="text-center text-white"
                  >
                    Error 404 
                    <br />
                     No encontramos esta ruta
                     
                    <span
                      aria-label="emoji"
                      role="img"
                    >
                      ðŸ™…
                    </span>
                  </h2>
                </Col>
              </Row>,
              "tag": "div",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <Col
                  md="7"
                  tag="div"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  <h2
                    className="text-center text-white"
                  >
                    Error 404 
                    <br />
                     No encontramos esta ruta
                     
                    <span
                      aria-label="emoji"
                      role="img"
                    >
                      ðŸ™…
                    </span>
                  </h2>
                </Col>,
                "className": "justify-content-center",
                "tag": "div",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <h2
                    className="text-center text-white"
                  >
                    Error 404 
                    <br />
                     No encontramos esta ruta
                     
                    <span
                      aria-label="emoji"
                      role="img"
                    >
                      ðŸ™…
                    </span>
                  </h2>,
                  "md": "7",
                  "tag": "div",
                  "widths": Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ],
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Error 404 ",
                      <br />,
                      " No encontramos esta ruta",
                      " ",
                      <span
                        aria-label="emoji"
                        role="img"
                      >
                        ðŸ™…
                      </span>,
                    ],
                    "className": "text-center text-white",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Error 404 ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    " No encontramos esta ruta",
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "aria-label": "emoji",
                        "children": "ðŸ™…",
                        "role": "img",
                      },
                      "ref": null,
                      "rendered": "ðŸ™…",
                      "type": "span",
                    },
                  ],
                  "type": "h2",
                },
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
